app/src/main/java/at/roteskreuz/stopcorona/model/repositories/NearbyRepository.kt

```diff
@@ -144,7 +144,7 @@ class NearbyRepositoryImpl(
 
     override val messageListener: MessageListener = object : MessageListener() {
 
-        override fun onFound(message: Message?) {
+        override fun onFound(message: Message) {
```

Comment:
Unfortunately, the source is java, which means we cannot be sure if the type is nullable or not. This change might give us NullPointerException and btw, the code below prints warnings about unnecessary safe call `?.`.